### PR TITLE
fixed missing translation, add undersocre js translations and version bump

### PR DESCRIPTION
**Pull request by umarmughal824**
It's a cherry-pick of this PR https://github.com/open-craft/problem-builder/pull/219.

- Fix translations for all the language
- Updated Test Cases which were failing
- Add underscorejs translations

### CODE REVIEW

**Comment by umarmughal824** [comment]
@xitij2000 @giovannicimolin I have cherry-picked those **problem-builder-v2** merged and reviewed changes here. Please review it.

**Comment by umarmughal824** [comment]
@giovannicimolin please run CircleCI test on this code.

File: setup.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 # Constants #########################################################
 
-VERSION = '3.1.5'
+VERSION = '3.1.6'
```
Comment (xitij2000) [inline setup.py]: Bump this to 3.2

**Comment by xitij2000** [comment]
@umarmughal824 The tests are running  now but I expect them to fail with the same issue as the other PR since the main issue causing the fails still needs to be addressed.

**Comment by umarmughal824** [comment]
@xitij2000 It's also update. 
- CIrcleCI tests passed
- Version bump

**Comment by umarmughal824** [comment]
@giovannicimolin please have a look on it too.

**Comment by xitij2000** [comment]
@umarmughal824 Please fix all the issues you fixed in the original PR here, and thoroughly test it before pinging for a review. I am immideately getting an error in the block. I'm pasting the relevant bits of the log here:
```
edx.devstack.lms     |   File "/edx/src/xblocks/problem-builder/problem_builder/step.py", line 290, in _render_view
edx.devstack.lms     |     }, i18n_service=self.i18n_service))
edx.devstack.lms     | TypeError: render_template() got an unexpected keyword argument 'i18n_service'
```

To be clear, test this on the hawthorn or master [devstack for vanilla edx-platform](https://github.com/edx/devstack). 

Also note that just recently the version of problem-builder used by the solutions fork has been bumped to v3. So v2 will likely no longer be supported.

**Comment by umarmughal824** [comment]
@xitij2000 I might have forgotten to cherry-pick one of the commits. Lets me verify it.

**Comment by umarmughal824** [comment]
@xitij2000 yes one commit was missed while cherry-pick process.

**Comment by umarmughal824** [comment]
@xitij2000 I have checked it out. Its ready to be reviewed.

**Comment by giovannicimolin** [comment]
@umarmughal824 I'll test and review the code, meanwhile, please update the description and the PR title to better show what these commits are doing, for a future reference.

**Comment by umarmughal824** [comment]
@giovannicimolin sure.

**Comment by umarmughal824** [comment]
@xitij2000 please have a look on it. It's already approved by @giovannicimolin.

**Comment by umarmughal824** [comment]
@xitij2000 will you please review it?

**Comment by umarmughal824** [comment]
@xitij2000 please create release tag its needed on ugrent bases.

**Pull request merged**